elements/pfe-codeblock/README.md

```diff
@@ -0,0 +1,72 @@
+# PatternFly Element | Codeblock element
+Render code in a styled and fromatted way
+
+## Usage
+Describe how best to use this web component along with best practices.
+
+```html
+<pfe-codeblock>
+    <!-- Default slot -->
+    <h2>This is pfe-codeblock</h2>
+    
+</pfe-codeblock>
+```
+
+### Accessibility
+Explain how this component meets accessibility standards.
+
+## Slots
+
+- `namedSlot`: Describe each available slot and best practices around what markup it can contain.
+
+## Attributes
+
+- `pfe-codeLanguage`: Describe this attribute and what function is serves.
+- `pfe-codeLineNumbers`: Describe this attribute and what function is serves.
+- `pfe-codeLineNumberStart`: Describe this attribute and what function is serves.
+- `pfe-codeTheme`: Describe this attribute and what function is serves.
+
+## Variable hooks
+
+Available hooks for styling:
+
+| Variable name | Default value | Region |
+| --- | --- | --- |
+| `--pfe-pfe-codeblock--Color` | `#252527` | N/A |
+
+## Events
```

Comment:
I don't think the component is doing anything with these events. We either need to update the detail or remove them.